docs/source/gbd2019_models/risk_effects/mm_risk_effects/index.rst

```diff
@@ -68,40 +64,40 @@ In Phase 2, some of these risk factors will become correlated with treatment in
     - Source
     - Note
   * - Sex
-    -
-    -
-    - Male
+    - 0.86 (0.68, 1.1)
+    - 0.83 (0.67, 1.03)
     - Female
+    - Male
     - Age, treatment(?)
-    -
-    -
-    -
+    - None 
```

Comment:
I'm not sure which of these this applies to, but where the source is RCT and treatment was not assigned based on the risk factor, I think this column should say "Treatment (randomized)".

Reply:
"This column" = "Adjustment variables"